src/main/java/com/diamondmarket/orders/service/DeleteOrdersService.java

```diff
@@ -0,0 +1,38 @@
+package com.diamondmarket.orders.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.diamondmarket.orders.model.Orders;
+import com.diamondmarket.orders.model.TransactionContext;
+import com.diamondmarket.orders.model.exceptions.DeleteOrderException;
+import com.diamondmarket.orders.model.exceptions.GetOrdersException;
+import com.diamondmarket.orders.repository.OrderRepository;
+
+@Service
+public class DeleteOrdersService {
+	
+	private OrderRepository orderRepository;
+	
+	@Autowired
+	public DeleteOrdersService(OrderRepository orderRepository) {		
+			this.orderRepository = orderRepository;
+			// TODO Auto-generated constructor stub
+	}
+	
+	public void deleteOrders(TransactionContext context, String orderId) throws DeleteOrderException {
+		 Orders findOne = orderRepository.findOne(orderId);
```

Comment:
it should be "Orders order" instead of  "Orders findOne".

Reply:
done